### PR TITLE
CRM-15771 - On 'submit credit card contribution' option to choose Auth N...

### PR DESCRIPTION
**Pull request by atif-shaikh**
...et is not present

https://issues.civicrm.org/jira/browse/CRM-15771

### CODE REVIEW

File: CRM/Member/Page/Tab.php [hunk 1]
```diff
@@ -316,7 +316,7 @@ public function run() {
     $this->preProcess();
 
     // check if we can process credit card membership
-    $newCredit = CRM_Core_Config::isEnabledBackOfficeCreditCardPayments();
+    $newCredit = CRM_Core_Config::isEnabledBackOfficeCreditCardPayments(CRM_Core_Config::isEnabledBackOfficeCreditCardPayments(CRM_Utils_Request::retrieve('mode', 'String', CRM_Core_DAO::$_nullObject, FALSE, NULL, 'GET')));
```
Comment (yashodha) [inline CRM/Member/Page/Tab.php]: This doesn't look right. Can you re-check?

File: CRM/Event/Page/Tab.php [hunk 2]
```diff
@@ -154,7 +154,7 @@ public function run() {
     $this->preProcess();
 
     // check if we can process credit card registration
-    $this->assign('newCredit', CRM_Core_Config::isEnabledBackOfficeCreditCardPayments());
+    $this->assign('newCredit', CRM_Core_Config::isEnabledBackOfficeCreditCardPayments(CRM_Core_Config::isEnabledBackOfficeCreditCardPayments(CRM_Utils_Request::retrieve('mode', 'String', CRM_Core_DAO::$_nullObject, FALSE, NULL, 'GET'))));
```
Comment (yashodha) [inline CRM/Event/Page/Tab.php]: Can you recheck?

**Comment by eileenmcnaughton** [comment]
OK - will take a look shortly

**Comment by eileenmcnaughton** [comment]
I did an alternate PR - see ticket

**Pull request closed without merging**